serde/__init__.py

```diff
@@ -16,7 +16,7 @@
     'is_deserializable',
     'from_dict',
     'from_tuple',
-    'asdict',
-    'astuple',
+    'to_dict',
+    'to_tuple',
```

Comment:
I actually never used the dataclasses `asdict` & `astuple` functions.
I just found it strange that `asdict` and `to_dict` from master basically did the same thing and wondered why `to_dict` was not exported under `serde`.

My main reason for renaming was to simplify the code and make the serializing & deserializing code more similar (have more symmetry).
For me it felt more natural that the inverse of `from_dict` is `to_dict` especially because the functions of all the other formats follow the same naming schema with `to_` & `from_`.

Reply:
Thanks for the explanation! I am curious of other people's opinion 🙂 

@adsharma @jfuechsl @andreymal  @alexmisk @pranavvp10 
There is a proposal to renaming `asdict`/`astuple` to `to_dict`/`to_tuple`
What do you guys think? 🤔